src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -339,6 +339,12 @@ public static FormulaValue Text(FormattingInfo formatInfo, IRContext irContext,
             if (args.Length > 1 && args[1] is StringValue fs)
             {
                 formatString = fs.Value;
+
+                if (!string.IsNullOrEmpty(formatString) && !TextFormatUtils.AllowedListToUseFormatString.Contains(args[0].Type._type))

```

Comment:
>IsNullOrEmpty [](http://example.com/codeflow?start=28&length=13)

Can the format string ever be null?

Reply:
i think no, i use it to check empty string.